.env.example

```
@@ -1,10 +1,19 @@
-SENTRY_DSN=sendry_dsn_url
+# Encryption secret for API keys decryption
+ENCRYPTION_SECRET=encryption_secret_key
+
+# Supabase database configuration
+NEXT_PUBLIC_SUPABASE_URL=http://127.0.0.1:54321
+NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=supabase_publishable_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)